docs/results.md

```diff
@@ -0,0 +1,376 @@
+The results for Performance test runs can be found in the [`results/perf`](https://github.com/puppetlabs/gatling-puppet-load-test/tree/master/results/perf) directory.
+
+The results for each Performance test run are contained within a separate timestamped sub-directory of the `results/perf` directory.
+The name of each result directory consists of the label `PERF_` to indicate a Performance run followed by a timestamp generated at the start of the run.
+For example:
+```
+results/perf
+└── PERF_1572446968
+```
+
+Performance result directories contain multiple files and sub-directories, each of which will be explained below.
+For example:
+  
+```
+results/perf/PERF_1572446968
+├── 20191030T144928Z-20191030T184941Z.tar.gz
+├── current_tune_settings.json
+├── end_epoch
+├── ip-10-227-0-214.amz-dev.puppet.net
+├── ip-10-227-0-248.amz-dev.puppet.net
+├── puppet-metrics-collector
+├── puppetserver.average.csv
+├── puppetserver.average.csv.html
+├── puppetserver.csv
+├── puppetserver.csv.html
+└── start_epoch
+```
+
+## Host directories
+The results directory contains sub-directories for each PE infrastructure host.
+In the Standard reference architecture example above this would be the PE 'master'.
+A sub-directory for the 'metric' node (where the Gatling simulation is run) is also included.
+The name of each sub-directory corresponds to the hostname of the node.
+
+### Master
+#### atop measurements
+The 'master' node directory contains the atop measurements provided by [beaker-benchmark](https://github.com/puppetlabs/beaker-benchmark) and additional CSV and HTML files generated from these files.
+For example:
+```
+results/perf/PERF_1572446968/ip-10-227-0-214.amz-dev.puppet.net
+├── atop_log_applestoapples_json.csv
+├── atop_log_applestoapples_json.detail.csv
+├── atop_log_applestoapples_json.detail.csv.html
+├── atop_log_applestoapples_json.log
+├── atop_log_applestoapples_json.log.tar.gz
+├── atop_log_applestoapples_json.summary.csv
+├── atop_log_applestoapples_json.summary.csv.html
+├── atop_log_warmupjit_json.log
+└── atop_log_warmupjit_json.log.tar.gz
+```
+
+In the above example the `atop_log_warmupjit_json.log` and `atop_log_applestoapples_json.log` files are the full atop logs produced by beaker-benchmark.
+The 'warmupjit' log is created during the 'warm-up' portion of the test run; the 'applestoapples' log is created during the main portion of the test run.
+Due to potentially large log file sizes beaker-benchmark compresses these files before copying to the test runner; these are the corresponding `tar.gz` files.
+
+The `atop_log_applestoapples_json.csv` file is a CSV file containing data extracted from the corresponding log file.
+It contains a summary of the measurements along with per-process measurements.
+However, the formatting of this file is technically invalid due to a mismatched number of columns per row between these two sections.
```

Comment:
More detail might be helpful here.  This file is two set of CSV data combined into a single file.  This caught me by surprise.  IMHO, this is also the core as to why it  is an invalid CSV file which cannot be corrected.

Reply:
Updated to add a bit more detail, although I didn't want to go into the weeds about beaker-benchmark since we're moving away from it soon.